app/liquid_tags/slideshare_tag.rb

```diff
@@ -1,31 +1,36 @@
 class SlideshareTag < LiquidTagBase
   PARTIAL = "liquids/slideshare".freeze
+  REGISTRY_REGEXP = %r{https://(?:www.)?slideshare.net/slideshow/embed_code/key/(?<id>\w{12,14})}
+  VALID_ID_REGEXP = /\A(?<id>\w{12,14})\Z/
+  REGEXP_OPTIONS = [REGISTRY_REGEXP, VALID_ID_REGEXP].freeze
 
-  def initialize(_tag_name, key, _parse_context)
+  def initialize(_tag_name, input, _parse_context)
     super
-    @key = validate(key.strip)
+
+    stripped_input = strip_tags(input)
+    @key = parse_input(stripped_input)
   end
 
   def render(_context)
     ApplicationController.render(
       partial: PARTIAL,
       locals: {
         key: @key,
-        height: 450
+        height: 487
```

Comment:
Not a blocker, but what's the background on this change from 450 to 487?

Reply:
@jeremyf Great question! This is the largest height value offered when generating an embed code from the slideshare website. The other options are 422, 397, and 292 (there was no 450). So I decided to go with the largest.